oauth2client/_helpers.py

```diff
@@ -26,11 +26,6 @@
 from six.moves import urllib
 
 
-__author__ = (
-    'rafek@google.com (Rafe Kaplan)',
-    'guido@google.com (Guido van Rossum)',
-)
-
```

Comment:
These guys are getting removed from here but aren't in contributors section, not sure if intentional?